os/drivers/Makefile

```diff
@@ -98,6 +99,7 @@ include usbhost$(DELIM)Make.defs
 include video$(DELIM)Make.defs
 include wireless$(DELIM)Make.defs
 
+CSRCS += kernel_test$(DELIM)kernel_test_drv.c
```

Comment:
I don't agree this change. That file should be handled in Make stubs of kernel_test.

Reply:
corrected